docs/recipe_09.rst

```diff
@@ -0,0 +1,314 @@
+.. _recipe_09:
+
+======================================
+Reading and Writing ORCA Files
+======================================
+
+- ``import cctk`` is assumed.
+- *cctk* was originally designed with Gaussian in mind, but supports basic writing of ORCA input files and parsing of ORCA output
+- Note that the MiniPrint option may make Orca output files unreadable by cctk
```

Comment:
Can you be specific and say what level of printing you recommend?  Then you can say you do NOT recommend using MiniPrint.  Are we checking the print level and warning or raising an error?